### PR TITLE
QTM-454 - New HTML Structure

### PR DESCRIPTION
**Pull request by alizeleal**
https://jirasoftware.catho.com.br/browse/QTM-454

### CODE REVIEW

**Review by MarcosViniciusPC — APPROVED** [review]
I have three suggestions for you that may be of great help to you.

**Code indentation:** Indenting helps you maintain a good hierarchy of elements in the html and contributes to good readability of the code. Doing it by hand is quite laborious and takes a lot of time. With that, Visual Studio Code plugins like prettier can save you this task and keep your code much prettier. 

How to make:
- Install "Prettier - Code formatter" in VS Code
- Open the file you want to indent/format and click with the mouse options button and select "Format Document With..." and in the opened window select "Prettier - Code formatter"
- Done, your code will be beautiful!

**Unused code:** Following good programming practices, do not leave unused code in your projects however much it may be used in the future. Only write necessary code and leave code that will be used in the future for the future. Context: I saw that there are several classes defined in the html elements that are not contained in the css files and this is not a good practice, because whoever revises it may think that it was a writing error instead of being something purposeful thought for the future. Additionally, these codes cause unnecessary performance issues. In the future maybe you'll change your mind and opt for an alternative solution instead of the solution you started doing (this happens to me often lol)

**Be specific in reviewing the assignment:** For example, on the page you created what do you expect me to look out for? The styling of new elements? The html structure? Code writing review? Or both? In the pr of catho-components or quantum there is a topic called Review Guide that we usually use to say what needs to be revised in the task.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+        <div class="TopBar" alt="Barra Superior de Navegação">
+
+            <!-- Logo -->
+            <span class="TopLogo">
+                <a href="#" class="CathoLogo" alt="Logotipo da Catho"><img src="imgs/cathoLogo.svg" alt="Logotipo da Catho"></a>
+            </span>
+
+            <!-- Main Navigation Menu -->
+            <span>
```
Comment (rapahaeru) [inline index.html]: Why did you use a `<span>` here?

A `span`, is usually for phrases or texts and inline elements.
For cases where your intention is a "block", i would suggest using a `div`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+</head>
+
+<body>
+
+    <!-- Header -->
+
+    <header>
+
+        <div class="TopBar" alt="Barra Superior de Navegação">
```
Comment (rapahaeru) [inline index.html]: `alt` prop does not exist in div element, only in `img` as a alternative text that describes the image content

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+            <!-- Logo -->
+            <span class="TopLogo">
+                <a href="#" class="CathoLogo" alt="Logotipo da Catho"><img src="imgs/cathoLogo.svg" alt="Logotipo da Catho"></a>
```
Comment (rapahaeru) [inline index.html]: same here, change the `alt` element for `title` and see in browser when you focus in the logo

File: index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            </span>
+
+            <!-- Main Navigation Menu -->
+            <span>
+                <nav class="NavigationBar" alt="Menu de Navegação">
```
Comment (rapahaeru) [inline index.html]: same here, alt element only used in `img` element

File: index.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                    <ul class="NavigationMenu">
+                        <li><a href="#">Catho Components</a></li>
+                        <li><a href="#">Quantum</a> </li>
+                    </ul>
+                </nav>
+            </span>
+
+            <!-- Hidden Navigation Menu -->
+            <div>
```
Comment (rapahaeru) [inline index.html]: Why we need this `div` element here?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        <div class="MainContent">
+
+        </div>
+
+    </main>
+
+    <!-- Footer -->
+
+    <footer>
+        <div class="FooterInformation">
```
Comment (rapahaeru) [inline index.html]: how about of put that class in `footer` element?
That way, we dont need this `div` element

File: index.html [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+            <p>Essa tela foi especialmente desenvolvida para a dev.Ela maravilhosa da ACME.</p>
+            <p>9 de fevereiro de 2023</p>
```
Comment (rapahaeru) [inline index.html]: how about of use a `date` element here?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

File: index.html [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+            <p>Por designer com mão de android</p>
+        </div>
+    </footer>
+
+    <!-- Scripts -->
+
+    <script src="#"></script>
```
Comment (rapahaeru) [inline index.html]: Do we need it, now?

**Review by rapahaeru — CHANGES_REQUESTED** [review]
How about of create a folder to concentrate all the statics files?
static_src/
... css/*
index.html

Remember that all this files will be deleted in future.

File: index.html (hunk 8 above)
Comment (alizeleal) [reply index.html]: It was for future reference, but I'll be removig for now

File: static_files/index.html [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+    <!-- Footer -->
+
+    <footer>
+      <p>
+        Essa tela foi especialmente desenvolvida para a dev.Ela maravilhosa da
+        ACME.
+      </p>
+      <p>
+        <date>9 de fevereiro de 2023</date>
```
Comment (rapahaeru) [inline static_files/index.html]: how about of:

```
            <p>
                <time datetime="2023-02-09">9 de fevereiro de 2023</time>
            </p>
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**